### PR TITLE
Dbscan gui fix

### PR DESCRIPTION
**Pull request by zacsimile**
The real fix for #199

### CODE REVIEW

File: PYME/recipes/localisations.py [hunk 1]
```diff
@@ -426,6 +426,25 @@ def execute(self, namespace):
     def hide_in_overview(self):
         return ['columns']
 
+    @property
+    def default_view(self):
+        import wx
```
Comment (David-Baddeley) [inline PYME/recipes/localisations.py]: What is this doing here?

Comment (David-Baddeley) [reply PYME/recipes/localisations.py]: IE we shouldn't need to test for wx.App

Comment (zacsimile) [reply PYME/recipes/localisations.py]: Can delete. Was following the example you told me to use as a template, `recipes.tablefilters`, line 71.

Comment (David-Baddeley) [reply PYME/recipes/localisations.py]: sorry, the ExtractTableChannel one would probably have been a better example. Have just made a few tweaks (and about to make a couple more).

**Comment by David-Baddeley** [comment]
PS - should really document what the pipeline_view is about - if not immediately obvious, this is what gets shown in the pipeline view on the left in VisGUI.

**Pull request merged**